### PR TITLE
Root name will always be transform to lower case.

### PR DESCRIPTION
**Pull request by pomek**
Fixes: #450

### CODE REVIEW

File: tests/view/document/document.js [hunk 1]
```diff
@@ -173,7 +173,7 @@ describe( 'Document', () => {
 			const domH1 = document.createElement( 'h1' );
 
 			const viewDocument = new Document();
-			viewDocument.createRoot( 'div' );
+			viewDocument.createRoot( 'DIV' );
```
Comment (Reinmar) [inline tests/view/document/document.js]: You shouldn't touch an existing test. Unless it's clearly incorrect or a new, refactored tests will check this cases as well, an existing test shouldn't be touched.

**Comment by Reinmar** [comment]
At least http://tests.ckeditor.dev:1030/build/amd/tests/ckeditor5/editor/standardeditor is failing. Please run all the tests.

**Comment by pomek** [comment]
@Reinmar, (Tests of) StandardEditor works fine for me…

image

I see another issue:

image

But I wasn't sure whether it is my fault. So, I checked behaviour of these test on master. After each changing branch, I also restarted gulp task and cleared cache:

- [UndoCommand on `t/450`](https://cloud.githubusercontent.com/assets/2270764/16220723/d16f9d28-378e-11e6-9aa2-b7c5b75f4843.png)
- [UndoCommand on `master`](https://cloud.githubusercontent.com/assets/2270764/16220734/e5f6b204-378e-11e6-987d-635dfcf3f345.png)
- [UndoEngine - Integration on `t/450`](https://cloud.githubusercontent.com/assets/2270764/16220705/b97ad1c4-378e-11e6-9c41-250c75651de4.png)
- [UndoEngine - Integration on `master`](https://cloud.githubusercontent.com/assets/2270764/16220761/19f83db6-378f-11e6-83e2-dff673df6dfe.png)

**Comment by pomek** [comment]
All tests work fine.

image

**Comment by scofalik** [comment]
It seems that everything is fine.

**Comment by scofalik** [comment]
Everything works fine on `master` if you have newest `ckeditor5-undo/master` branch,

**Pull request merged**